### ISSUE TITLE
./hack/test-integration.sh fails on my mac

### ISSUE THREAD

**Issue by roberthbailey** [GoogleCloudPlatform/kubernetes#7282]
I'm not sure when this started, but if I run `./hack/test-integration.sh` it fails:

```
W0423 23:23:36.658577   84822 master.go:196] Portal net unspecified. Defaulting to 10.0.0.0/24.
I0423 23:23:36.658788   84822 master.go:226] Will report xxxx::xxxx:xxx:xxxx:xxxx as public IP address.
I0423 23:23:36.749171   84822 publish.go:152] setting endpoints for master service "kubernetes-ro" to &{{ } {kubernetes-ro  default    0001-01-01 00:00:00 +0000 UTC <nil> map[] map[]} [{[{fe80::6203:8ff:fea0:6324 <nil>}] [{ 7080 TCP}]}]}
E0423 23:23:36.750885   84822 publish.go:70] Can't create ro endpoints: Endpoints "kubernetes-ro" is invalid: subsets[0].addresses[0].ip: invalid value 'xxxx::xxxx:xxx:xxxx:xxxx': invalid IPv4 address
I0423 23:23:36.751216   84822 publish.go:152] setting endpoints for master service "kubernetes" to &{{ } {kubernetes  default    0001-01-01 00:00:00 +0000 UTC <nil> map[] map[]} [{[{fe80::6203:8ff:fea0:6324 <nil>}] [{ 6443 TCP}]}]}
E0423 23:23:36.752301   84822 publish.go:45] Can't create rw endpoints: Endpoints "kubernetes" is invalid: subsets[0].addresses[0].ip: invalid value 'xxxx::xxxx:xxx:xxxx:xxxx': invalid IPv4 address
FAIL
FAIL	github.com/GoogleCloudPlatform/kubernetes/test/integration	17.562s
!!! Error in /Users/robertbailey/Work/go/src/github.com/roberthbailey/kubernetes/hack/test-go.sh:115
  'go test "${goflags[@]:+${goflags[@]}}" ${KUBE_RACE} ${KUBE_TIMEOUT} "${@+${@/#/${KUBE_GO_PACKAGE}/}}"' exited with status 1
Call stack:
  1: /Users/robertbailey/Work/go/src/github.com/roberthbailey/kubernetes/hack/test-go.sh:115 runTests(...)
  2: /Users/robertbailey/Work/go/src/github.com/roberthbailey/kubernetes/hack/test-go.sh:208 main(...)
Exiting with status 1
!!! Error in ./hack/test-integration.sh:36
  'KUBE_GOFLAGS="-tags 'integration no-docker' " KUBE_RACE="-race" KUBE_TEST_API_VERSIONS="$1" "${KUBE_ROOT}/hack/test-go.sh" test/integration' exited with status 1
Call stack:
  1: ./hack/test-integration.sh:36 runTests(...)
  2: ./hack/test-integration.sh:61 main(...)
Exiting with status 1
+++ [0423 23:23:36] Integration test cleanup complete
```

The same tests pass when running as part of `make release` but I find that for debugging test failures it's much nicer to be able to run them in isolation.

**Comment by smarterclayton** [comment]
Oh, good, I fixed both accidentally. :)

**Comment by roberthbailey** [comment]
Fixed by #7721.
